### ISSUE TITLE
Disqus blocked

### ISSUE THREAD

**Issue by rugk** [EFForg/privacybadgerfirefox#536]
PrivacyBadger blocks the CDN of Disqus  (here already unblocked) and so it does not load the comments at all:
![pp_disqus](https://cloud.githubusercontent.com/assets/11966684/9388234/81dd8934-4764-11e5-84c6-42eeec1d29b4.png)

Disqus displays this message:
![disquserror](https://cloud.githubusercontent.com/assets/11966684/9388365/20e96cf0-4765-11e5-91a6-fdfcac61f739.PNG)

**Comment by nyuszika7h** [comment]
Duplicate of #469.
